Dockerfile

```diff
@@ -1,18 +1,11 @@
-ARG GIT_COMMIT=unspecified
-ARG GIT_REMOTE=unspecified
 ARG VERSION=unspecified
 
 FROM python:3.9-alpine
 
-ARG GIT_COMMIT
-ARG GIT_REMOTE
 ARG VERSION
 
-LABEL git_commit=${GIT_COMMIT}
-LABEL git_remote=${GIT_REMOTE}
-LABEL maintainer="mark.feldhousen@trio.dhs.gov"
-LABEL vendor="Cyber and Infrastructure Security Agency"
-LABEL version=${VERSION}
+LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
```

Comment:
In case folks want to add more authors later, it might be good to have a comment here stating whether this string should be a comma-delimited list of email addresses or some other format.

Reply:
I've added comments to both places that generate labels for the container.
The [annotation](https://github.com/opencontainers/image-spec/blob/master/annotations.md) keys and values are defined in the [OCI Image Format Specification](https://github.com/opencontainers/image-spec#oci-image-format-specification).

That specific annotation, `org.opencontainers.image.authors`, is defined as `freeform string`: 
![tenor](https://user-images.githubusercontent.com/3229435/97900044-22757900-1d08-11eb-9874-b28dbde87aec.gif)